frontend/package.json

```
@@ -17,6 +17,9 @@
     "generate-env": "node scripts/generate-env.js"
   },
   "dependencies": {
+    "@fuels/connectors": "^0.43.2",
+    "@hookform/resolvers": "^5.2.2",
```

Biome (2.1.2)

[error] The key @hookform/resolvers was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)